Dockerfile

```diff
@@ -20,8 +20,6 @@ EXPOSE 8081
 
 CMD npm run start
 
-FROM base as tracing
-
 RUN npm install dd-trace --save
```

Comment:
Should we move the install to before the npm run start? Maybe to line 14 after `npm run ci`?